pkg/controllers/controllers.go

```diff
@@ -35,20 +41,30 @@ func init() {
 	metrics.MustRegister() // Registers cross-controller metrics
 }
 
-func GetControllers(ctx operator.Context, cluster *state.Cluster, settingsStore settingsstore.Store, cloudProvider cloudprovider.CloudProvider) []controller.Controller {
-	provisioner := provisioning.NewProvisioner(ctx, ctx.KubeClient, ctx.Clientset.CoreV1(), ctx.EventRecorder, cloudProvider, cluster, settingsStore)
+func NewControllers(
+	ctx context.Context,
+	clock clock.Clock,
+	kubeClient client.Client,
+	clientset kubernetes.Interface,
```

Comment:
`kubeInterface`?

Reply:
Maybe some consistent name for `kubernetes.Interface` would be good

Reply:
Ah -- I meant to make this kubernetesInterface everywhere. It brainlessly concatenate package and type.